Source/WebCore/dom/PointerEvent.cpp

```diff
@@ -128,9 +167,22 @@ PointerEvent::PointerEvent(const AtomString& type, PointerID pointerId, const St
 
 PointerEvent::~PointerEvent() = default;
 
-Vector<Ref<PointerEvent>> PointerEvent::getCoalescedEvents()
+Vector<Ref<PointerEvent>> PointerEvent::getCoalescedEvents() const
 {
     return m_coalescedEvents;
 }
 
+void PointerEvent::receivedTarget()
+{
+    MouseRelatedEvent::receivedTarget();
+
+    // When a trusted PointerEvent's target is changed, user agents SHOULD, for each event in the
+    // coalesced events list and predicted events list:
+    //
+    //  Set the event's target to match the target of the "parent" pointer event.
```

Comment:
I don't think this comment is necessary. This is just the standards text and doesn't explain why we did this, and the code is super readable here.